src/relay.cpp

```diff
@@ -71,6 +71,153 @@ std::map<int, std::string> counterMap = {{DHCPv6_MESSAGE_TYPE_UNKNOWN, "Unknown"
 /* interface to vlan mapping */
 std::unordered_map<std::string, std::string> vlan_map;
 
+
+/* Class Options */
+
+bool Options::Add(OptionCode key, const uint8_t *value, uint16_t len) {
+    std::vector<uint8_t> option_v(value, value + len);
+    options[key] = option_v;
+    return true;
+}
+
+bool Options::Delete(OptionCode key) {
+    if (options.find(key) == options.end()) {
+        return false;
```

Comment:
What is the reason for returning false here?
What is the harm of deleting non-existing key?

Reply:
Delete non-existing key is no harm, I make it return false in order for a notification to callers in future. 
This place return false will not impart option unmarshal/marshal operation.